Lab2/Lab2.iml

```diff
@@ -7,9 +7,108 @@
       <sourceFolder url="file://$MODULE_DIR$/src/main/java" isTestSource="false" />
```

Comment:
You really don't need to import *.iml, as it is a config file created by your IDE. This one will be recreated if someone clones and opens your project with the same IDE, and may cause problems for those who use any other IDE (for example Eclipse)